src/azure-cli/azure/cli/command_modules/reservations/_help.py

```diff
@@ -162,3 +162,84 @@
     type: string
     short-summary: Id of the subscription to look up applied reservations
 """
+
+helps['reservations reservation-order calculate'] = """
+type: command
+short-summary: calculate price
+long-summary: calculate price for reservation
+parameters:
+  - name: --applied-scope-type
+    type: string
+    short-summary: choose Single or Shared as appliedscope
+  - name: --applied-scopes
+    type: string
+    short-summary: Subscription that the benefit will be applied. Do not specify if --applied-scope-type is Shared.
```

Comment:
Do you have a [validator](https://github.com/Azure/azure-cli/blob/8adf97355a605d147afc26f6c1b6f6afb6651c2f/doc/authoring_command_modules/authoring_commands.md#validators) to check applied-scope-type and --applied-scopes value and throw accurate error messages?
In addition, this parameter only supports subscription level?

Reply:
Should this be handled in our code base? whatever customer passed into this command, we will validate those value in our side.

Reply:
Actually you can validate in CLI side, but it's fine if service will validate and give correct error message. In this way, what will happen if both two arguments are provided?

Reply:
I have put it as an enum in CLI. Somehow if you put anything other than Single or Shared. A message will be provided:
az reservations reservation-order calculate: 'SingleShared' is not a valid value for '--applied-scope-type'. See 'az reservations reservation-order calculate --help'.

Reply:
Also in our side, we will handle this as well